pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -142,7 +138,21 @@ func (Plugin) OnInboundRouteConfiguration(in *plugin.InputParams, routeConfigura
 	default:
 		log.Warn("Unknown listener type in mixer#OnOutboundRouteConfiguration")
 	}
+}
+
+func buildMixerOpaqueConfig(check, _ /*forward*/ bool, destinationService string) *mccpb.ServiceConfig {
+	out := &mccpb.ServiceConfig{
+		// Report calls are never disabled. Forward is currently removed.
```

Comment:
How does mixer find source service without forwarding attributes?

Reply:
DisableForward is just not available, meaning (I think) that forwarding attributes cannot be disabled but are always present. The comment was confusing, I've updated it.